packages/requisitions/src/SupplierRequisition/api/operations.graphql

```diff
@@ -0,0 +1,124 @@
+mutation insertRequestRequisition(
+  $storeId: String!
+  $input: InsertRequestRequisitionInput!
+) {
+  insertRequestRequisition(input: $input, storeId: $storeId) {
+    ... on RequisitionNode {
+      __typename
+      id
+    }
+  }
+}
+
+mutation updateRequestRequisition(
+  $storeId: String!
+  $input: UpdateRequestRequisitionInput!
+) {
+  updateRequestRequisition(input: $input, storeId: $storeId) {
+    ... on RequisitionNode {
+      __typename
+      id
+    }
+  }
+}
+
+# mutation deleteSupplierRequisitions($ids: [DeleteSupplierRequisitionInput!]) {
+#   batchSupplierRequisition(deleteSupplierRequisitions: $ids) {
+#     __typename
+#     deleteSupplierRequisitions {
+#       __typename
+#       id
+#     }
+#   }
+# }
+
+fragment RequestRequisitionLine on RequisitionLineNode {
+  id
+  itemId
+  requestedQuantity
+  calculatedQuantity
+  itemStats {
+    averageMonthlyConsumption
+    stockOnHand
+    monthsOfStock
+  }
+}
```

Comment:
not sure about this, but not bad to gen the type from this subset of fields. Though, the type to use is `RequestRequisitionLineFragment` (and the same for the other fragments below) - I hate the suffix. Can use a config option to remove *all* suffixes, which would take a bit of time to rename the whole code base (i..e  `requisitionQuery` would be `requisition`). I also think fighting against this has always just ended up in more time spent, so thought to just go with it 🤷